### ISSUE TITLE
Buy: Water Filter

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#286]
as noted in https://github.com/dwyl/home/issues/284#issuecomment-2296974981 the `@home` water dispenser 💧 
(integrated into the Samsung Fridge/Freezer https://github.com/dwyl/home/issues/82) 
is dispensing water _very_ slowly ... 🐌 ⏳ 

This is the original that came with the fridge: 

![image](https://github.com/user-attachments/assets/2f356ac3-67a8-4cd4-b4be-3c83936663a7)

We _could_ just buy another of these from `Samsung` ... 
***Or*** a white-label compatible one. 

@iteles thoughts?

**Comment by iteles** [comment]
## Originals

The Amazon ones look very cheap on the surface, but the shipping is €24 😅 
https://www.amazon.es/-/pt/dp/B0C2Z8TTQ1

<img width="1406" alt="image" src="https://github.com/user-attachments/assets/7bb485c1-b4f9-4ae7-ac65-cb0605ff2b2c">

They're ultimately the same price as these, but these will be here a week earlier and then we can claim IVA back I think:
https://www.filtrosfrigorificosamericanos.pt/pack-3-filtros-externos-originais-geladeira-samsung-da29-10105j-p-202.html
<img width="1428" alt="image" src="https://github.com/user-attachments/assets/175a7ec9-4590-4c10-b557-56d43a2405bc">

## Compatibles

These are the best value by a [loooong way](https://www.filtrosfrigorificosamericanos.pt/pack-2-filtros-de-geladeira-compativeis-samsung-da2910105j-p-207.html) and also have good reviews:
https://www.amazon.es/dp/B07DCQHS6W
<img width="1427" alt="image" src="https://github.com/user-attachments/assets/7d6c48db-55d0-4b6c-b51c-cc187619cde1">

The only question is whether we buy originals or compatibles?

I would be tempted to try the compatibles just to see whether the water quality is the same and how long they last.

But my concern is that the ice maker is playing up - if the filter is not the culprit and it's still being weird, then the repair people might put it down to a non-original filter instead of actually repairing it and we'll waste the €30 call-out charge 🙄 

Go original this time and test compatibles next time?

**Comment by nelsonic** [comment]
100% go original to avoid issues with service. 👌
But buy a compatible immediately so we have it in-stock to swap when needed.

**Comment by nelsonic** [comment]
Pack of 3 filters arrived today:
![IMG_7131](https://github.com/user-attachments/assets/53205c45-fcec-4e8f-b01b-85be754de501)

I’ve got one out and am installing it now:
![IMG_7132](https://github.com/user-attachments/assets/bcfcf9a9-fa23-40e0-a4cf-2c72045165e2)

**Comment by nelsonic** [comment]
Replaced the water filter; old and new side-by-side:

![IMG_7134](https://github.com/user-attachments/assets/bd3374a4-682b-4eab-baaf-9dcde10671ab)

Yes, those are *wet* paper/kitchen towels; I spaced and forgot to turn off the tap before disconnecting the filter hose/line so there was a bit of water … 😬
(Which I cleaned up with mostly cloth towel but a bit of paper towel too…) 

So if you ever need to change the filter, remember to full turn the right tap under the north-facing sink:

![IMG_7135](https://github.com/user-attachments/assets/ea4980d5-be83-4329-b5dc-b8d48618125d)

Do this *before* attempting to disconnect the hoses from the filter. 😉

Anyway, cleaned up the excess water, connected the hoses and turned the tap back on. Everything works as expected and the water flows marginally faster. 👌


Also, related: the ice maker is full:

![IMG_7136](https://github.com/user-attachments/assets/fb581625-4dbc-4f7e-8f60-af54ae383f61)

So I would consider postponing the visit from the Samsung Fridge technician. 🧑‍🔧💭
